ruby-challenges.rb

```diff
@@ -22,13 +37,56 @@
 beatles_album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
+def removed_vowels(string)
+    string.gsub(/[aeiou]/, '') 
+end
+
+p removed_vowels(beatles_album1)
+p removed_vowels(beatles_album2)
+p removed_vowels(beatles_album3)
+
+beatles_album1 = 'Rubber Soul'
+# Expected output: 'Rbbr Sl'
+beatles_album2 = 'Sgt Pepper'
+# Expected output: 'Sgt Pppr'
+beatles_album3 = 'Abbey Road'
+# Expected output: 'bby Rd'
+
+# ---> "Rbbr Sl"
+# "Sgt Pppr"
+# "Abby Rd"
 
 
 # -------------------3) Create a method that takes in a string and checks if the string is a palindrome. A palindrome is the same word spelled forward or backward. Use the test variables provided.
+# psuedocode:
+# Create a method that takes in a string and checks if the string is a palindrome. 
+# input: string
+# output: string
+# will reverse the strings with .reverse and use if statement to check to see if the oriiginal string == the reversed string  
 
 palindrome_tester1 = 'Racecar'
 # Expected output: 'Racecar is a palindrome'
 palindrome_tester2 = 'LEARN'
 # Expected output: 'LEARN is not a palindrome'
 palindrome_tester3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+
+def is_or_not(string)
+    if string == string.reverse
+        p "#{string} is a palindrome"
+    else 
+        p "#{string} is not a palindrome"
+    end
+end
```

Comment:
Similarly to the previous challenge, make sure you are taking capital letters into account! If you plugged in "Rotator" into your method, it is comparing "Rotator" to "rotatoR", which would come out to be false, which would return that it is not a palindrome, even though it is